src/Components/Api.js

```diff
@@ -0,0 +1,13 @@
+const BASE_URL = `https://bootcamp-api.codeit.kr/`;
```

Comment:
api까지 추가를 해도 무리가 없을듯하긴 합니다~!
```suggestion
const BASE_URL = `https://bootcamp-api.codeit.kr/api`;
```